src/components/ArrowKeyFocusManager.js

```diff
@@ -26,7 +26,7 @@ class ArrowKeyFocusManager extends Component {
         const arrowDownConfig = CONST.KEYBOARD_SHORTCUTS.ARROW_DOWN;
 
         this.unsubscribeArrowUpKey = KeyboardShortcut.subscribe(arrowUpConfig.shortcutKey, () => {
-            if (this.props.maxIndex < 1) {
```

Comment:
@roryabraham I need help in confirming this check. Any specific reason we used 1 earlier? It causes an issue when we've got only one element in the list. Hence I've changed to 0.

Reply:
I am pretty skeptical of this change – there are a lot of regression tests that need to be done locally for this PR. I am going to post them in the issue description.

Reply:
Sorry, maybe it's just my eyes, but to me none of those items look preselected / highlighted.

My knee-jerk reaction here was that on web + desktop the first item should be highlighted, but on iOS/Android/mWeb probably not. After thinking about it some more though, this is a power-user feature and therefore none of the options need to be pre-selected on any platform, because doesn't need to be immediately obvious that you can use the keyboard to navigate these list items.

So here's what I think we should do for the expected behavior:

- When you open the menu, nothing is highlighted.
- If you press ArrowUp at this point, the bottom item will be highlighted. If you press ArrowDown, the top item will be highlighted.
- After an option is highlighted, the list should cycle such that:
    - If the top item is highlighted and you press ArrowUp, then the bottom item is highlighted
    - The the bottom item is highlighted, and you press ArrowDown, then the top item is highlighted
    
Does that sound good? If not, let's chat in slack about it. Thanks!

Reply:
@roryabraham To fix that the it should allow you to select the when the there's only 1 item in the list, I updated the check for `maxIndex < 0`.